.prow.sh

```diff
@@ -19,6 +19,8 @@
 # Only these tests make sense for csi-driver-nfs until we can integrate k/k
 # e2es.
 : ${CSI_PROW_TESTS:="unit"}
+: ${CSI_PROW_E2E_ALPHA_GATES:="GenericEphemeralVolume=true,CSIStorageCapacity=true,PodSecurity=false"}
+: ${CSI_PROW_E2E_ALPHA_GATES_LATEST:="GenericEphemeralVolume=true,CSIStorageCapacity=true,PodSecurity=false"}
```

Comment:
@pohly I may be completely off here, neverthless would like to ask , how to disable a feature gate ex: `podSecurity=false` in prow cluster.  by default enabled `podSecurity` feature gate, causes the e2e nfs pod  we have in the test to go wrong due to the extra/caps or permissions it carry in its spec.. (https://github.com/kubernetes-csi/csi-driver-nfs/pull/332#issuecomment-1142281287) , so I am trying to disable the feature gate and run the e2e

Reply:
This is currently not supported for the non-alpha cluster. `start_cluster` for the non-alpha cluster would have to be passed some feature gate list here:
https://github.com/kubernetes-csi/csi-release-tools/blob/e4dab7ff57c24cf3e8d37cd3365636fddaff7e0a/prow.sh#L1257

You can add `CSI_PROW_E2E_GATES[_LATEST]` to your local prow.sh copy, set that here, and when it works, submit that change to csi-release-tools.

Reply:
hmmm.. it still fails on the same error [1] , I added a new flag to start_cluster() https://github.com/kubernetes-csi/csi-driver-nfs/pull/332/commits/7a35140746835102cba70abce6e20711380a8949 and it looks it is not picking up for some reason. 

[1]             ```  Message: "pods \"nfs-volume-tester-sc2rz\" is forbidden: violates PodSecurity \"restricted:latest\": allowPrivilegeEscalation != false (container \"volume-tester\" must set securityContext.allowPrivilegeEscalation=false), unrestricted capabilities (container \"volume-tester\" must set securityContext.capabilities.drop=[\"ALL\"]), runAsNonRoot != true (pod or container \"volume-tester\" must set securityContext.runAsNonRoot=true), seccompProfile (pod or container \"volume-tester\" must set securityContext.seccompProfile.type to \"RuntimeDefault\" or \"Localhost\")",
              Reason: "Forbidden",```
from https://storage.googleapis.com/kubernetes-jenkins/pr-logs/pull/kubernetes-csi_csi-driver-nfs/332/pull-csi-driver-nfs-e2e/1534843287561572352/build-log.txt

@pohly did I miss anything on sending the flag to prow job or parsing and using it in start_cluster?

Reply:
pull-kubernetes-csi-csi-driver-nfs doesn't use prow.sh for testing...

https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes-csi/csi-driver-nfs/csi-driver-nfs-unmanaged.yaml

Reply:
Hmmm.. Thanks !